addons/stock/models/stock_move.py

```diff
@@ -938,6 +938,10 @@ def _propagate_split(self, new_move, qty):
             new_move_prop = self.move_dest_id.split(qty)
             new_move.write({'move_dest_id': new_move_prop})
 
+    def _prepare_move_split_vals(self, defaults):
```

Comment:
This could contain the default data of lines 961-969, instead of returning itself.

Reply:
Are you sure? That mean we should give plenty of arguments to that function (not everything comes from self).

Reply:
Hum yeah indeed, not great. Ok then